reasoner/controller/ConditionController.java

```diff
@@ -45,11 +43,6 @@ public class ConditionController extends ConjunctionController<
         this.condition = condition;
     }
 
-    @Override
-    Set<Concludable> concludablesTriggeringRules() {
-        return condition.concludablesTriggeringRules(registry().conceptManager(), registry().logicManager());
-    }
-
```

Comment:
Removing this means the physical Resolvable instances returned by condition.concludablesTriggeringRules are different to the ones used by the reasoner.